value/sys.go

```diff
@@ -39,53 +43,134 @@ To convert a time vector to a seconds value:
 To print seconds in Unix date format:
```

Comment:
Need to add a one-line description of the new feature in sysHelp

Reply:
Done.